Core/Model/LineaTransferenciaStock.php

```diff
@@ -94,4 +99,57 @@ public static function tableName()
     {
         return 'lineastransferenciasstock';
     }
+
```

Comment:
class core/model/base/businessdocumentline.php has not been used as a guide. If you look at the class you will see that it is already responsible for knowing what the initial amount was. You do not need the controller to tell you.